src/js/stores/EventStore.jsx

```diff
@@ -21,153 +20,106 @@ class EventStore extends EventEmitter {
   }
 
   getStreams() {
-    $.ajax({
-      url: `${url}/stream`,
-      dataType: 'json',
-      headers: {
-        Authorization: `Bearer ${AuthService.getToken()}`,
-      },
-      cache: false,
+    AjaxService.get({
+      endpoint: 'stream',
       success(response) {
         const streams = response.data.map(d => d.attributes.stream);
         dispatcher.dispatch({ type: 'STREAM_GET', stream: streams });
       },
-      failure(error) {
+      error(error) {
         dispatcher.dispatch({ type: 'EVENTS_ERROR', error: error.error });
       },
     });
     return this.events;
   }
 
   getAll() {
-    $.ajax({
-      url: `${url}/event`,
-      dataType: 'json',
-      headers: {
-        Authorization: `Bearer ${AuthService.getToken()}`,
-      },
-      cache: false,
+    AjaxService.get({
+      endpoint: 'event',
       success(response) {
         const events = response.data;
-        $.ajax({
-          url: `${url}/user/${AuthService.getCurrentUserId()}/subscription`,
-          dataType: 'json',
-          headers: {
-            Authorization: `Bearer ${AuthService.getToken()}`,
-          },
-          cache: false,
+        AjaxService.get({
+          endpoint: `user/${AuthService.getCurrentUserId()}/subscription`,
           success(response2) {
             const subscription = response2.data.map(sub => sub.attributes.event_id);
             dispatcher.dispatch({ type: 'EVENTS_GET', events, subscriptions: subscription });
           },
-          failure(error) {
+          error(error) {
             dispatcher.dispatch({ type: 'EVENTS_ERROR', error: error.error });
           },
         });
       },
-      failure(error) {
+      error(error) {
         dispatcher.dispatch({ type: 'EVENTS_ERROR', error: error.error });
       },
     });
     return this.events;
   }
 
   updateEvent(event) {
-    $.ajax({
-      url: `${url}/event/${event.id}`,
-      data: JSON.stringify(event),
-      type: 'PATCH',
-      contentType: 'application/json',
-      processData: false,
-      dataType: 'json',
-      headers: {
-        Authorization: `Bearer ${AuthService.getToken()}`,
-      },
+    AjaxService.update({
+      endpoint: 'event',
+      data: event,
       success() {
-        dispatcher.dispatch({ type: 'EVENT_UPDATE', event });
+        dispatcher.dispatch({ type: 'EVENTS_UPDATE', event });
```

Comment:
Was this intentionally pluralized? There is still an `EVENT_UPDATE` case in the handleActions function.